dev/com.ibm.ws.security.oauth/src/com/ibm/ws/security/oauth20/web/ClientAuthentication.java

```diff
@@ -443,7 +444,8 @@ protected boolean validateResourceOwnerCredential(HttpServletRequest request, Ht
             throw e;
         } catch (Exception e1) {
             Tr.error(tc, "security.oauth20.endpoint.resowner.auth.error", userName);
-            throw new OidcServerException("invalid_resource_owner_credential", OIDCConstants.ERROR_SERVER_ERROR, HttpServletResponse.SC_BAD_REQUEST, e1);
+            throw new OidcServerException(new BrowserAndServerLogMessage(tc, "security.oauth20.endpoint.resowner.auth.error", userName), OIDCConstants.ERROR_INVALID_CLIENT_METADATA, HttpServletResponse.SC_BAD_REQUEST);
```

Comment:
I think this is one we don't want to change. This particular error message is indicating that a username or password is incorrect, which is actually information we probably don't want to leak to an end user. I'd leave this one as it was.